haiku-apps/catkeyseditor/catkeyseditor-0.1.1.recipe

```diff
@@ -10,7 +10,10 @@ HOMEPAGE="https://github.com/puckipedia/BeLocalized"
 COPYRIGHT="2014-2015 Puck Meerburg"
 LICENSE="MIT"
 REVISION="1"
```

Comment:
Cool, thanks!
Do you want to approve your review before one of us do the merge?

BTW, I have already checked all the SHA256 checksums 1 or 2 days ago ;-)